.env

```diff
@@ -30,3 +30,4 @@ SMTP_REAL_NON_CANONICAL_DEFAULT=example.com
 BACKUP_S3_BUCKET=example-bkp
 BACKUP_EMAIL_FROM=backup@example.com
 BACKUP_EMAIL_TO=alerts@example.com
+TZ=Europe/Madrid
```

Comment:
But you're not using this variable anywhere, are you?

Reply:
You are right, I'll fix that.
I begin to understand more about docker-compose stuff :smiley:

Reply:
Done